WordPress/Classes/ViewRelated/NUX/FancyAlertViewController+LoginError.swift

```diff
@@ -1,21 +1,24 @@
 import UIKit
+import wpxmlrpc
 
 extension FancyAlertViewController {
-    static func siteAddressHelpController() -> FancyAlertViewController {
-        struct Strings {
-            static let titleText = NSLocalizedString("What's my site address?", comment: "Title of alert helping users understand their site address")
-            static let bodyText = NSLocalizedString("Your site address appears in the bar at the the top of the screen when you visit your site in Safari.", comment: "Body text of alert helping users understand their site address")
-            static let OK = NSLocalizedString("OK", comment: "Ok button for dismissing alert helping users understand their site address")
-            static let moreHelp = NSLocalizedString("Need more help?", comment: "Title of the more help button on alert helping users understand their site address")
-        }
+    struct Strings {
+        static let titleText = NSLocalizedString("What's my site address?", comment: "Title of alert helping users understand their site address")
+        static let bodyText = NSLocalizedString("Your site address appears in the bar at the the top of the screen when you visit your site in Safari.", comment: "Body text of alert helping users understand their site address")
+        static let OK = NSLocalizedString("OK", comment: "Ok button for dismissing alert helping users understand their site address")
+        static let moreHelp = NSLocalizedString("Need more help?", comment: "Title of the more help button on alert helping users understand their site address")
+    }
 
-        typealias Button = FancyAlertViewController.Config.ButtonConfig
+    typealias ButtonConfig = FancyAlertViewController.Config.ButtonConfig
```

Comment:
This is definitely a more logical name, but I was just copying what @frosty did.

ITS HIS FAULT.

*points*
*runs away*